pkg/dashboard/dashboard.go

```diff
@@ -78,3 +115,60 @@ func Dashboard(opts Options) http.Handler {
 		writeTemplate(tmpl, opts, &data, w)
 	})
 }
+
+func calculateContainerCost(costPerCPUFloat float64, costPerGBFloat float64, c summary.ContainerSummary) float64 {
+	var memCost, cpuCost float64
+	if c.Limits != nil {
+		cpuCost = costPerCPUFloat * (float64(c.Requests.Cpu().Value() + c.Limits.Cpu().Value())) / 2
+		memCost = costPerGBFloat * (ConvertToGB(c.Requests.Memory().Value()) + ConvertToGB(c.Limits.Memory().Value())) / 2
```

Comment:
This is a little more complicated--the user might set `Limits.CPU`, but not `Limits.Memory` (or vice versa). Requests can also be nil or 0.

I would write the code like this:
```go
cpuRequests := 0
memRequests := 0
cpuLimits := 0
memLimits := 0

if c.Limits != nil {
  cpuLimits = c.Limits.Cpu().Value()
  memLimits = c.Limits.Memory().Value()
}
if c.Requests != nil {
  cpuRequests = c.Requests.Cpu().Value()
  memRequests = c.Requests.Memory().Value()
}

func getNonZeroAverage(req, limit float64) float64 {
  avg := (req + limit) / 2.0
  if req == 0.0 {
    avg = limit
  }
  if limit == 0.0 {
    avg = request
  }
  return avg
}

cpuCost := costPerCPUFloat * getNonZeroAverage(cpuRequests, cpuLimits)
memCost := costPerGBFloat * getNonZeroAverage(memRequests, memLimits)
```

Reply:
I updated the code. Thank you for the suggestion!